__tests__/test1.js

```diff
@@ -1,19 +1,22 @@
 const Pet = require('../src/test1');
+// add a line break here
 describe('constructor', () => {
     it('returns an object', () => {
       expect(new Pet('Fido')).toBeInstanceOf(Object);
     });
     it ('checks for pet name', () => {
         expect(new Pet('Fido')).toHaveProperty('name', 'Fido')
+        // no line breaks before a block of code closes
 
     });
     it ('checks correct age', () => {
         expect(new Pet('Fido')).toHaveProperty('age', 0)
     })
 }
-)
+) // this should trail the closing bracket
```

Comment:
Wherever you have a callback function, the closing block and closing parentheses should be on the same line:

```js
myFunc('hello', () => {
  console.log('hi!');
})
```